docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+version: "3.5"
+
+networks:
+  metavoice-net:
+    driver: bridge
+
+volumes:
+  hf-cache:
+    driver: local
+
+services:
+  metavoice-server:
+    volumes:
+      - hf-cache:/.hf-cache
+      - ./assets:/app/assets
+    deploy:
+      replicas: 1
+      resources:
+        reservations:
+          devices:
+            - driver: nvidia
+              count: 1
+              capabilities: [gpu]
+    runtime: nvidia
+    ipc: host
+    tty: true # enable colorized logs
+    build:
+      context: .
+    command: ['--huggingface_repo_id=metavoiceio/metavoice-1B-v0.1', "--port=80"]
```

Comment:
why port 80? you are later mapping it to 8869?
also, when i run this yaml, the server logging is lost.

Reply:
thats up to the user to decide...